dockers/docker-base-bullseye/Dockerfile.j2

```diff
@@ -83,6 +83,18 @@ RUN pip3 install supervisord-dependent-startup==1.4.0
 
 RUN mkdir -p /var/log/supervisor /etc/supervisor/conf.d
 
+# Install gcc, libc6-dev and python3-dev for compiling python-lzf
+RUN apt-get -y install build-essential libc6-dev python3-dev
+
+# Install python-lzf
+RUN pip3 install 'python-lzf==0.2.4'
+
+# Install rdbtools
+RUN pip3 install 'rdbtools==0.1.15'
+
+# Uninstall gcc, libc6-dev and python3-dev for compiling python-lzf
+RUN apt-get -y remove build-essential libc6-dev python3-dev
```

Comment:
>python3 [](http://example.com/codeflow?start=48&length=7)

I think these packages has many dependencies, and remove them may leave the dependencies inside the docker image for no use. Could you use `dpkg -l` to dump before and after your change, and then compare the package list? How large is the image size increasing by new python packages?